### ISSUE TITLE
Issue with my dashboard and donwloading resume

### ISSUE THREAD

**Issue by MWN09** [AmruthPillai/Reactive-Resume#1422]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues

### Product Variant

Cloud (http://rxresu.me)

### Current Behavior

Taking too much time to load my dashboard but doesn't show images on the different versions of resumes.

If I open my resume and try to download it, after 5 minutes it gives an error and says to raise this issue here

### Expected Behavior

![image](https://github.com/AmruthPillai/Reactive-Resume/assets/93926763/9a7cda62-1991-4d38-b722-9b74fdef2aa8)

**Comment by VanBrugg3n** [comment]
Same issue here. I'm using the cloud instance through Firefox. My dashboard doesn't even load.

**Comment by MWN09** [comment]
> Nevermind, it somehow fixed itself. Weird.

Yeah, It hangs for a while and then works again. Mostly stuck after 9 pm (according to Berlin time). But that's annoying :(

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
